quickstart/www/authoring/authoring-activities.php

```diff
@@ -0,0 +1,65 @@
+<?php
+    /**
+     * Copyright (c) 2017 Learnosity, MIT License
+     *
+     * Basic example of embedding Author API for creating Activities,
+     * using `mode: "activity_list"`.
+     */
+
+    // - - - - - - Section 1: server-side configuration - - - - - - //
+
+    // Include server side Learnosity SDK.
+    require_once __DIR__ . '/../../src/vendor/autoload.php';
+    use LearnositySdk\Request\Init as LearnosityInit;
```

Comment:
```suggestion
    use LearnositySdk\Request\Init;
```

Reply:
Updated. I couldn't use your commit for some reason, so I replicated it manually.